### PR TITLE
Redirects browse

### PR DESCRIPTION
**Pull request by jz575**
- [x] Login and Register activities link to each other
- [x] User can register/login
- [x] Register/Login redirect to browse communities
- [x] Miscellaneous styling

### CODE REVIEW

**Review by Strevia — APPROVED** [review]
Looks good, works as expected.

**Review by eleferrets — COMMENTED** [review]
All good!

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '7.2.2' apply false
```
Comment (eleferrets) [inline build.gradle:3]: You wanted to update Gradle to get the notifications to stop?

File: app/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-/build
\ No newline at end of file
+/build
+secrets.xml
```
Comment (eleferrets) [inline app/.gitignore:2]: Technically, the secrets were already in the main gitignore but this was closer

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -54,7 +61,7 @@
         <!-- make sure within the application tag, otherwise app will crash with XmlResourceParser errors -->
         <provider
             android:name="androidx.core.content.FileProvider"
-            android:authorities="com.codepath.mxerfileprovider"
+            android:authorities="com.codepath.mxer.fileprovider"
```
Comment (eleferrets) [inline app/src/main/AndroidManifest.xml:64]: This was a better solution to that issue

**Review by eleferrets — APPROVED** [review]

**Pull request merged**